src/libpgexporter/management.c

```diff
@@ -422,6 +423,15 @@ pgexporter_management_read_isalive(SSL* ssl, int socket, int* status)
 
    *status = pgexporter_read_int32(&buf);
 
+   if (status)
```

Comment:
Should I make this `if(status == 1)` or is it fine?

Reply:
We need to have the output on the `cli.c` side of things, and only if `-v` was passed

Reply:
Hmmm, I was taking `pgexporter_management_read_status` as a reference. Also....what should be the output according to you if `-v` is not passed?

Reply:
`management.c` shouldn't ideally have any output - the read functions should pass the values out to the caller... in some way.

For now, you can use `*status == 0`...

Reply:
> `management.c` shouldn't ideally have any output - the read functions should pass the values out to the caller... in some way.

Got it...so should I shift the output to [this place](https://github.com/pgexporter/pgexporter/blob/ff197819089d8a4bf7481a6f3075a6b7a86101b8/src/cli.c#L561)?

Also should I still go with printing the output for `-v` flag or should it be printed regardless?

> For now, you can use `*status == 0`...

Okay!

Reply:
Yeah, ideally - in `cli.c` with `-v`.

They are in `management.c` because it was easier for the `..._read_` signatures.

We can do that in another pull request...